configuration.md

```diff
@@ -0,0 +1,61 @@
+# Policy Controller Configuration 
+
+The policy controller is primarily configured through environment variables.  When running
+the policy controller as a Kubernetes pod, this is accomplished through the pod manifest `env`
+section.
+
+## The kube-policy-controller container
+
+### Etcd configuration
```

Comment:
Etcd Settings

We're not configuring etcd here, we're configuring calico with etcd's settings.